packages/element-api/package.json

```diff
@@ -45,6 +46,7 @@
     "eslint-plugin-node": "^8.0.0",
     "eslint-plugin-security": "^1.4.0",
     "faker": "^4.1.0",
+    "firebase-tools": "^7.3.2",
```

Comment:
Firebase-tools already installed here: https://github.com/decentralized-identity/element/blob/465799f6da01d7c84dd740705824c23f73924708/package.json#L21

Reply:
I think we don't want it at that level, ideally we don't even have it as an api dependency (its only needed for firebase deployment of express app, which is very thin.

I'm gonna remove it from root level.